Description: Remove annoying warning about the version of libxml used
Author: Ryan Niebur <ryanryan52@gmail.com>
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=546843
Forwarded: not-needed
Index: git/lib/nokogiri/version.rb
===================================================================
--- git.orig/lib/nokogiri/version.rb	2013-11-03 11:47:24.422149897 +0100
+++ git/lib/nokogiri/version.rb	2013-11-03 11:48:37.348681267 +0100
@@ -36,11 +36,7 @@
     def warnings
       return [] unless libxml2?
 
-      if compiled_parser_version != loaded_parser_version
-        ["Nokogiri was built against LibXML version #{compiled_parser_version}, but has dynamically loaded #{loaded_parser_version}"]
-      else
-        []
-      end
+      []
     end
 
     def to_hash
